frame/support/procedural/src/storage/instance_trait.rs

```diff
@@ -83,6 +87,7 @@ pub fn decl_and_impl(scrate: &TokenStream, def: &DeclStorageDefExt) -> TokenStre
 				/// instance.
 				#[doc(hidden)]
 			),
+			index: 0,
```

Comment:
What is "inherent_instance"???

I gave it index 0 too, but technically that conflicts with `DefaultInstance`...?

Maybe it should be 255?

Reply:
I would need to check the code, but somewhere we require an instance, even if we don't use any instancing. I think somewhere related to the genesis config.

Reply:
Based on the doc I would change this to 255. I am not sure when we need this instance though.

Reply:
Yeah, I'm right. 

This is only used to make the type system happy, so the actual index isn't that important.